scss/partials/_quotes.scss

```diff
@@ -73,15 +73,24 @@
       color: $primary-white;
       margin-top: 0;
     }
+
+    & .quote--title {
+      color: $secondary-green-2;
+    }
   }
 
   & > .quote__text {
```

Comment:
This block seems to have a lot of these nested selectors that creates an additional level of specificity on the block's elements and modifiers? Is this strictly necessary? It's a bit of an antipattern for BEM, so if we can avoid nested selectors like this, we should. If it's too much trouble to refactor now, let's leave a comment here to come back to it.

Reply:
This seems to be a lot since it requires changes and testing on the testimonial single and slider section sections. I've left a TODO note on it.